src/test/java/com/ksm/wstbackoffice/WstBackofficeApplicationTests.java

```diff
@@ -8,6 +8,6 @@
 
 	@Test
 	void contextLoads() {
```

Comment:
Please run all tests and make sure you have one test failed. Try to fix it, or ask me why it is failing

Reply:
done